ci-build.sh

```diff
@@ -31,7 +31,7 @@ execute 'Approving recipe quality' check_recipe_quality
 for package in "${packages[@]}"; do
     execute 'Building binary' makepkg --noconfirm --noprogressbar --nocheck --syncdeps --rmdeps --cleanbuild
     execute 'Building source' makepkg --noconfirm --noprogressbar --allsource
-    execute 'Installing' yes:pacman --noprogressbar --upgrade *.pkg.tar.*
+    grep -qFx "$(<../ci-dont-install-list.txt)" <<< "${package}" || execute 'Installing' yes:pacman --noprogressbar --upgrade *.pkg.tar.*
```

Comment:
interesting way of using the subshell+redirect to use a file as a list of exceptions. Haven't really seen this thing done at all.

Reply:
I'm not quite sure if that was a compliment or a criticism :smiley:

Reply:
had a 🤦 epiphany trying to fall asleep
```bash
grep -qFx "${package}" ../ci-dont-install-list.txt || ...
```
What was I thinking?

Reply:
[KISS](https://wiki.archlinux.org/index.php/Arch_terminology#KISS)